src/pages/guides/getting_started/tut.md

```diff
@@ -2,28 +2,28 @@
 keywords:
   - Adobe Express
   - Express Add-on SDK
-  - Express Editor
+  - Express Document API
+  - Authoring Sandbox
   - Adobe Express
   - Add-on SDK
   - SDK
   - JavaScript
   - Extend
   - Extensibility
   - API
-  - Document API
-title: Document API tutorial
-description: This is a Document API in-depth tutorial
+title: Adobe Express Document API tutorial
+description: This is an Adobe Express Document API in-depth tutorial
 contributors:
   - https://github.com/undavide
 ---
 
-# Document API tutorial
+# Adobe Express Document API tutorial
```

Comment:
I would change the name to something more descriptive since this is a tutorial.
For example:  Creating grids with Express add-ons using the Document API

Reply:
You're right, tutorials will benefit from a more descriptive title. I see that Ari Fuchs suggested "Building your first add-on with Document APIs", which I'm fine with—it shouldn't be singular, though, API? Maybe the other way around is correct, too, but perhaps it's less confusing for 3P.
(I get that terminology is something there's a lot of attention to get right, I'll keep that in mind for future work!)